src/debugger/BreakpointManager.js

```diff
@@ -25,6 +25,7 @@ export class BreakpointManager {
   _breakpointMaps: Map<string, PerFileBreakpointMap>;
   _previousStop: StoppedData;
   _channel: DebugChannel;
+  _lastExecuted: StoppedData;
```

Comment:
Rename "StoppedData" to be SourceData. It does not necessary mean stopped state anymore.